src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -17,3 +28,65 @@ def read_well_trajectory(self) -> Tuple[Array, Array]:
         x = [entry["displacement"] for entry in self.input_content["trajectory"]["data"]]
         y = [-entry["vertical_depth"] for entry in self.input_content["trajectory"]["data"]]
         return Array(x, LENGTH_UNIT), Array(y, LENGTH_UNIT)
+
+    def read_tubing_materials(self) -> List[Dict[str, Union[Scalar, str]]]:
+        """ "Read the data for the tubings from SCORE input file"""
+        tubing_data = []
+        for section in self.input_content["operation"]["tubing_string"]["string_sections"]:
+            thermal_property = section["pipe"]["grade"]["thermomechanical_property"]
+            tubing_data.append(
+                {
+                    "name": section["pipe"]["grade"]["name"],
+                    "density": Scalar(thermal_property["density"], DENSITY_UNIT),
+                    "thermal_conductivity": Scalar(
+                        thermal_property["thermal_conductivity"], THERMAL_CONDUCTIVITY_UNIT
+                    ),
+                    "specific_heat": Scalar(thermal_property["specific_heat"], SPECIFIT_HEAT_UNIT),
+                    "thermal_expansion": Scalar(
+                        thermal_property["thermal_expansion_coefficient"], THERMAL_EXPANSION_UNIT
+                    ),
+                    "young_modulus": Scalar(thermal_property["e"], YOUNG_MODULUS_UNIT),
+                    "poisson_coefficient": Scalar(thermal_property["nu"], FRACTION_UNIT),
```

Comment:
I know that in portugues it is "Coeficiente de Poisson", check if it is in english as well.

Reply:
Good point. Ideed "Poisson Ratio" in english must be more correct.

Reply:
Fixed now